From: Duan Jiong <duanj.fnst@cn.fujitsu.com>
Date: Fri, 1 Nov 2013 13:09:43 +0800
Subject: vxlan: Use ERR_CAST inlined function instead of ERR_PTR(PTR_ERR(...))
Patch-mainline: v3.13-rc1
Git-commit: e50fddc8b0c0698cba446221be3654aa0a642372
References: bsc#938963 FATE#319084

trivial patch converting ERR_PTR(PTR_ERR()) into ERR_CAST().
No functional changes.

Signed-off-by: Duan Jiong <duanj.fnst@cn.fujitsu.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/net/vxlan.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/vxlan.c
+++ b/drivers/net/vxlan.c
@@ -2403,7 +2403,7 @@ static struct vxlan_sock *vxlan_socket_c
 		sock = create_v4_sock(net, port);
 	if (IS_ERR(sock)) {
 		kfree(vs);
-		return ERR_PTR(PTR_ERR(sock));
+		return ERR_CAST(sock);
 	}
 
 	vs->sock = sock;
